src/main.rs

```diff
@@ -67,6 +67,10 @@ fn run() -> Result<()> {
     debug!("[main] log level: {}", log_level);
     debug!("[main] custom config path: {:?}", custom_config);
 
+    if let Ok(_matched) = comp_matches(build_app(), &matches) {
```

Comment:
By doing so you ignore all potential errors. You need to keep the `?` operator so that error are handled properly. Sth like:

```suggestion
    if comp_matches(build_app(), &matches)? {
```